index.js

```diff
@@ -136,6 +137,7 @@ var avro_decoded_data = function (schema_data, record) {
   const type = avro.parse(schema_data)
   var decoded = new Buffer(record, 'base64')
   var verify = type.fromBuffer(decoded)
+  console.log(decoded)
```

Comment:
You should also added more information to this log so you know where it's coming from when you see all the logs together.